serving/backend/server.py

```diff
@@ -19,9 +19,8 @@
 
 @app.post("/make_prediction")
 async def make_prediction(properties: Properties):
-    ordered_properties = ["cha", "con", "dex", "int", "str", "wis", "ac", "hp"]
     properties_dict = properties.dict()
-    stats = {p: properties_dict[p] for p in ordered_properties}
-    level = calculate_level(monster_stats=stats, model=model)
+    properties_dict.pop("name")
```

Comment:
To w sumie po co front przesyła ten name do backendu?

Reply:
Będzie usunięte przy PR frontu dla cf